src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -469,10 +466,6 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
           width: 52px;
         }
 
-        :host([narrow]) .weather-icon {
```

Comment:
Same here

Reply:
Fixed by restricting the selector to `:host([narrow]) .icon-image .weather-icon` so it only applies if it's the main icon and not a forecast icon.